docs/source/release_notes.rst

```diff
@@ -22,7 +23,7 @@ Future Release
         * Run lint fix on latest dependency update pull requests (:pr:`1640`, :pr:`1641`)
 
     Thanks to the following people for contributing to this release:
-    :user:`gsheni`
+    :user:`gsheni`, :user:`bchen1116`
```

Comment:
@ParthivNaresh , you should be on here as well right?

Reply:
@Cmancuso Ah yes my bad